webapp/src/app.tsx

```diff
@@ -1843,7 +1844,7 @@ export class ProjectView
         return this.state.running;
     }
 
-    runSimulator(opts: compiler.CompileOptions = {}, clickTrigger?: boolean) {
+    runSimulator(opts: compiler.CompileOptions = {}, clickTrigger?: boolean, cancellationToken?: pxt.Util.CancellationToken) {
```

Comment:
Can we have a run option object that is a superset of compileoptions? This is getting ugly.

Reply:
I think I can just move them all into compiler.CompileOptions